decision_service/dsclientlib/dsclientlib.vcxproj.filters

```diff
@@ -0,0 +1,105 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup>
+    <Filter Include="Source Files">
+      <UniqueIdentifier>{4FC737F1-C7A5-4376-A066-2A32D752A2FF}</UniqueIdentifier>
+      <Extensions>cpp;c;cc;cxx;def;odl;idl;hpj;bat;asm;asmx</Extensions>
+    </Filter>
+    <Filter Include="Header Files">
+      <UniqueIdentifier>{93995380-89BD-4b04-88EB-625FBE52EBFB}</UniqueIdentifier>
+      <Extensions>h;hh;hpp;hxx;hm;inl;inc;xsd</Extensions>
```

Comment:
do we need this file?

Reply:
Helps organize files in visual studio.  Any reason to remove it?

---
In reply to: [186733204](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r186733204) [](ancestors = 186733204)

Reply:
aren't those auto-generated?

---
In reply to: [186865764](https://github.com/JohnLangford/vowpal_wabbit/pull/1479#discussion_r186865764) [](ancestors = 186865764,186733204)